### PR TITLE
[LOOP-5153] Remove HealthKit dependency from LoopAlgorithm

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/LOOP-5153

### CODE REVIEW

File: Common/HKUnit.swift [hunk 1]
```diff
@@ -6,25 +6,17 @@
...
-        if self == HKUnit.millimolesPerLiter {
+        if self == LoopUnit.millimolesPerLiter {
             return NSLocalizedString("mmol/L", comment: "The short unit display string for millimoles of glucose per liter")
         } else if self == .milligramsPerDeciliter {
             return NSLocalizedString("mg/dL", comment: "The short unit display string for milligrams of glucose per decilter")
-        } else if self == .internationalUnit() {
+        } else if self == .internationalUnit {
             return NSLocalizedString("U", comment: "The short unit display string for international units of insulin")
-        } else if self == .gram() {
+        } else if self == .gram {
```
Comment (nhamming) [inline Common/HKUnit.swift]: [nit] is this needed? Or can this be defined in LoopAlgorithm?

File: LibreDemoPlugin/LibreDemoCGMManager.swift [hunk 2]
```diff
@@ -7,11 +7,12 @@
 //
 
 import Foundation
+import HealthKit
```
Comment (nhamming) [inline LibreDemoPlugin/LibreDemoCGMManager.swift:10]: [nit] was it intended to add HealthKit here?

**Review by nhamming — APPROVED** [review]
Couple of non-blocking questions. LGTM

File: LibreDemoPlugin/LibreDemoCGMManager.swift (hunk 2 above)
Comment (Camji55) [reply LibreDemoPlugin/LibreDemoCGMManager.swift:10]: I just reordered it. Still being used below!

File: Common/HKUnit.swift (hunk 1 above)
Comment (Camji55) [reply Common/HKUnit.swift]: Moved :)

**Pull request merged**